graylog2-server/src/main/java/org/graylog2/inputs/InputServiceImpl.java

```diff
@@ -174,6 +224,16 @@ public void addStaticField(Input input, final String key, final String value) th
     }
 
     @Override
+    public Input disable(Input input) {
+        return null;
+    }
+
+    @Override
+    public Input enable(Input input) {
```

Comment:
What are the `enable` and `disable` methods for? I guess you added them to handle the input state, but I ask anyway.